lib/logger/index.ts

```diff
@@ -123,6 +125,20 @@ loggerLevels.forEach((loggerLevel) => {
   logger.once[loggerLevel] = logOnceFn as never;
 });
 
+if (is.string(process.env.LOG_FILE)) {
+  validateLogLevel(process.env.LOG_FILE_LEVEL);
```

Comment:
can we return the validated level here and use it below, so we don't need the unsafe cast?